openjdk/build.xml

```diff
@@ -53,12 +53,12 @@
 			<contains string="${SPEC}" substring="zos" />
 			<then>
 				<exec executable="tar" failonerror="true">
-					<arg line="xfo jtreg-4.2.0-tip.tar -C ${DEST}" />
+					<arg line="xfo jtreg-4.2.0-tip.tar" />
 				</exec>
 			</then>
 		<else>
-			<exec executable="tar" failonerror="true">
-				<arg line="xf jtreg-4.2.0-tip.tar -C ${DEST}" />
+			<exec executable="sh" failonerror="true">
```

Comment:
Worth commenting why we're not using the tar task

Reply:
I think we'd have to comment on it in lots of places :-) I'm trying to do the minimal required here to give me something that works without being inconsistent with the rest of the file in order to validate this change on it's own.

Reply:
Also, there's no indication in https://docs.ansible.com/ansible/latest/collections/ansible/builtin/unarchive_module.html that it can handle an equivalent of the strip operation